scripts/gha/build_testapps.py

```diff
@@ -195,15 +184,18 @@ def main(argv):
   else:
     timestamp = ""
 
+  config = config_reader.read_config()
+
+  github_repo = False
```

Comment:
This variable is misleading - it will only be set to True if we're building from repo **and** building for iOS. If someone decides to add any logic that depends on whether we're building from repo for Desktop or Android, they'll run into problems.

It would make more sense to move the check out of _generate_makefiles_from_repo, and only run that method if github_repo is true and we're building for iOS. We should also log it (whether we've detected that we're building from repo) for debugging purposes.